activerecord/test/cases/insert_all_test.rb

```diff
@@ -335,6 +335,11 @@ def test_upsert_all_works_with_partitioned_indexes
                  Measurement.where(city_id: 2).pluck(:logdate, :peaktemp, :unitsales)
   end
 
+  def test_insert_all_with_enum_values
+    Book.insert_all [{ status: :published, isbn: "1234566" }, { status: :proposed, isbn: "1234567" }]
+    assert_equal Book.where(isbn: ["1234566", "1234567"]).pluck(:status), ['published', 'proposed']
```

Comment:
I'm not sure why rubocop did not run, but we use double quotes style for string literal, and use `assert_equal expected, actual` style like as other assertions.
And also, the result of `pluck` without `order` is not always stable order (especially on PostgreSQL), add `order(:id)`.

```suggestion
    assert_equal ["published", "proposed"], Book.where(isbn: ["1234566", "1234567"]).order(:id).pluck(:status)
```